generic_pva.xml.fdm_material

```diff
@@ -40,8 +40,11 @@ Generic PVA profile. The data in this file may not be correct for your specific
            <setting key="standby temperature">160</setting>
            <setting key="print cooling">0.0</setting>
            <hotend id="0.25 mm" />
+               <setting key="hardware compatible">yes</setting>
            <hotend id="0.4 mm" />
+               <setting key="hardware compatible">yes</setting>
            <hotend id="0.8 mm" />
+               <setting key="hardware compatible">yes</setting>
```

Comment:
I think you wanted these to be put inside the `hotend` tags, but the `hotend` tags close themselves with the `/>` suffix, so these `setting` tags are all in the same `machine` parent tag now and this makes (two of) them duplicates.